src/cameras/CubeCamera.d.ts

```diff
@@ -5,7 +5,12 @@ import { Object3D } from './../core/Object3D';
 
 export class CubeCamera extends Object3D {
 
-	constructor( near?: number, far?: number, cubeResolution?: number );
+	constructor(
+		near?: number,
+		 far?: number,
+		 cubeResolution?: number,
+		 options?: WebGLRenderTargetOptions
```

Comment:
You also have to add the following line at the top of the file. Otherwise `WebGLRenderTargetOptions` is not defined.

```ts
import { WebGLRenderTargetOptions } from './../renderers/WebGLRenderTarget';
```

Reply:
Yep, sorry about that, trying for a fly-by PR and going too fast :/